bin/cozy-light

```diff
@@ -88,16 +94,17 @@ if (!process.argv.slice(2).length) {
 
 // Manage errors
 
-process.on('uncaughtException', function (err) {
+process.on('uncaughtException', function(err){
   if (err) {
-    logger.warn('An exception is uncaught');
-    logger.raw(err);
-    logger.raw(err.stack);
-    actions.exit();
+    cozyLight.logger.warn('An exception is uncaught');
+    cozyLight.logger.raw(err);
+    cozyLight.logger.raw(err.stack);
   }
+  cozyLight.stop(cozyLight.exit);
```

Comment:
We should not do that. If an app runned by Cozy Light crashes, it will crash the whole platform.